spec/features/profile_spec.rb

```diff
@@ -28,8 +28,8 @@
     end
 
     it "should not have option to remove account" do
-      page.should_not have_content("Remove account")
-      current_path.should == profile_account_path
+      expect(page).not_to have_content("Remove account")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.